build_script.py

```diff
@@ -81,11 +81,17 @@ def main():
                         action="store_true",
                         dest="test",
                         default=False)
+    parser.add_argument("--arch",
+                        help="target architecture",
+                        action="store",
+                        dest="arch",
+                        default=None)
```

Comment:
You don't need to explicitly mark `default=None` here. `argparse` already uses `None` in the absence of the option being specified.